store/file/file.go

```diff
@@ -46,6 +44,11 @@ type fileStore struct {
 }
 
 func (m *fileStore) Init(opts ...store.Option) error {
+	m.options = store.Options{}
```

Comment:
Do not blow away the options. Only some of them may be included in the update.

Reply:
Cheers I was actually thinking about this